app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardGeneralStep/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Field, reduxForm } from 'redux-form';
+import { required } from 'redux-form-validators';
+import { Form } from 'patternfly-react';
+import { FormField } from '../../../formUtilities/FormField';
+
+const MappingWizardGeneralStep = props => (
```

Comment:
I think the "component" itself can be split into "MappingWizardGeneralStep.js"... but fine w/ this for now... we have the concept of "connected" inside index.js and the "unconnected" outside... This makes testing easier (if you see Overview tests )

Reply:
I agree with this, leaving a note here so I remember to address it if I beat @AparnaKarve to it.

Reply:
Will look into this shortly